transaction/sort.go

```diff
@@ -37,6 +43,10 @@ func CompareTxnsByFee(txn1, txn2 *Transaction) int {
 	return 0
 }
 
+func IsLowFeeTxn(txn *Transaction) bool {
+	return txn.UnsignedTx.Fee < config.Parameters.LowTxnFee
+}
+
```

Comment:
`IsLowFeeTxn` was defined but no one used it.

Reply:
It's reversed in case we need to change back in the future